gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Sun Nov 01 21:40:16 SGT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```

Comment:
concerning, any reason why?

Reply:
Huh might be because I deleted everything and git cloned the whole tp again

Reply:
Idk what difference this makes lmao

Reply:
try reverting this particular chunk and see if you can still run?